### PR TITLE
Umpqua bank

### PR DESCRIPTION
**Pull request by kevbob**
Adding Umpqua bank

### CODE REVIEW

File: _data/banking.yml [hunk 1]
```diff
@@ -475,6 +475,12 @@ websites:
       img: triodosbank.png
       tfa: No
 
+    - name: Umpqua Bank
+      url: https://www.umpquabank.com/
+      twitter: umpquabank
+      img  umpqua.png
```
Comment (Carlgo11) [inline _data/banking.yml]: you forgot an `:` :wink:

File: _data/banking.yml [hunk 2]
```diff
@@ -475,6 +475,12 @@ websites:
       img: triodosbank.png
       tfa: No
 
+    - name: Umpqua Bank
+      url: https://www.umpquabank.com/
+      twitter: umpquabank
```
Comment (Carlgo11) [inline _data/banking.yml]: I suggest you use `askumpquabank` instead.

**Comment by Carlgo11** [comment]
Hello @kevbob,
I suggest you use this image instead. It's way smaller.
umpqua

**Comment by kevbob** [comment]
Thanks for catching that! Used the suggested icon.

**Comment by kevbob** [comment]
Oh, hey- I did not use askumpquabank as suggested. I guess I could change that.  I thought about it deliberately, and just went to the main twitter account. To be sure, they respond with the ask account when you talk to their main corporate account, so it probably makes no difference, but making the change now.

**Comment by Carlgo11** [comment]
Thanks for the quick edits! :)

**Comment by jamcat22** [comment]
Awesome job @kevbob!

While your changes have been merged, it can take some time before they show up on the website. More info about this process is available on our [wiki](https://github.com/2factorauth/twofactorauth/wiki/FAQ_MERGE).

Thanks!

**Pull request merged**